runtime.md

```diff
@@ -15,9 +15,9 @@ This MUST be unique across all containers on this host.
 There is no requirement that it be unique across hosts.
 * **`status`**: (string) is the runtime state of the container.
 The value MAY be one of:
-  * `created` : the container has been created but the user-specified code has not yet been executed
-  * `running` : the container has been created and the user-specified code is running
-  * `stopped` : the container has been created and the user-specified code has been executed but is no longer running
+  * `created` : the container process has neither exited nor executed the user-specified code
+  * `running` : the container process has executed the user-specified code but has not exited
```

Comment:
meaning of `user-specified code` vs `container process` here seems confusing to me. For example, if `container process` is some init piece of the runtime, does this mean that the init piece has to be run for the lifetime of the user process because `stopped` only happens when the container process has exited ? Or did I understand this incorrectly ?

What do you think about replacing `user-specified code` with `container process` completely. Something like

```
* `created` : the container has been created but the container process has not yet been executed.
* `running`: the container process has been executed but has not exited.
* `stopped`: the container process has exited.
```